plugins/CoreAdminHome/lang/en.json

```diff
@@ -19,7 +19,8 @@
         "ImageTrackingLink": "Image Tracking Link",
         "ImportingServerLogs": "Importing Server Logs",
         "ImportingServerLogsDesc": "An alternative to tracking visitors through the browser (either via JavaScript or an image link) is to continuously import server logs. Learn more about %1$sServer Log File Analytics%2$s.",
-        "InvalidPluginsWarning": "The following plugins are not compatible with %1$s and could not be loaded: %2$s.",
+        "InvalidPluginsWarning": "The following plugins could not be loaded: %1$s.",
+        "InvalidPluginsWarningReasons": "Possible reasons of this situation are: plugin is not compatible with %1$s, plugin's directory is not readable or is empty.",
```

Comment:
Maybe rather something like 

`Please check if the plugin's directory exists and is readable. If so, the plugin may not be compatible with %1$s`.

Problem is people won't know how to check whether a plugin is compatible. It would be even better to check all these things and then directly point it out. Eg we could check whether a plugin is not compatible, we could check if directory does not exist and mention it, and we could check if directory exists but is empty (or misses a plugin.json)